dll/cpl/sysdm/lang/ru-RU.rc

```diff
@@ -324,33 +324,33 @@ BEGIN
 END
 
 IDD_VISUALEFFECTS DIALOGEX 6, 18, 227, 204
-CAPTION "Visual Effects"
+CAPTION "Визуальные эффекты"
 FONT 8, "MS Shell Dlg"
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
 BEGIN
-   LTEXT "Select the settings you want to use for the appearance and performance of ReactOS on this computer.", IDC_STATIC, 7, 6, 208, 18
+   LTEXT "Выберите настройки, которые вы хотите использовать для внешнего вида и производительности ReactOS на этом компьютере.", IDC_STATIC, 7, 6, 208, 18
    CONTROL "", IDC_TREE, "SysTreeView32", TVS_LINESATROOT | TVS_DISABLEDRAGDROP | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP, 7, 32, 213, 168
 END
 
 IDD_ADVANCEDPERF DIALOGEX 6, 18, 227, 128
-CAPTION "Advanced"
+CAPTION "Дополнительно"
 FONT 8, "MS Shell Dlg"
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
 BEGIN
-   GROUPBOX "CPU scheduling", IDC_STATIC, 7, 8, 213, 46
-   LTEXT "Adjust for best performance of:", IDC_STATIC, 14, 23, 199, 8
-   AUTORADIOBUTTON "&Programs", IDC_CPUCLIENT, 14, 36, 75, 12
-   AUTORADIOBUTTON "Background &services", IDC_CPUSERVER, 96, 36, 85, 12
-   GROUPBOX "Virtual memory", IDC_STATIC, 7, 60, 213, 57
-   LTEXT "A paging file is an area on the hard disk that ReactOS uses as if it were RAM.", IDC_STATIC, 14, 74, 199, 16
-   PUSHBUTTON "&Change...", IDC_CHANGESWAP, 163, 97, 50, 14
+   GROUPBOX "Распределение времени процессора", IDC_STATIC, 7, 8, 213, 46
+   LTEXT "Оптимизировать работу:", IDC_STATIC, 14, 23, 199, 8
+   AUTORADIOBUTTON "&Программ", IDC_CPUCLIENT, 14, 36, 75, 12
+   AUTORADIOBUTTON "&Служб, работающих в фоновом режиме", IDC_CPUSERVER, 96, 36, 85, 12
+   GROUPBOX "Виртуальная память", IDC_STATIC, 7, 60, 213, 57
+   LTEXT "Файл подкачки - это область на жестком диске используемая для хранения страниц виртуальной памяти.", IDC_STATIC, 14, 74, 199, 16
```

Comment:
```suggestion
   LTEXT "Файл подкачки - это область на жёстком диске, используемая для хранения страниц виртуальной памяти.", IDC_STATIC, 14, 74, 199, 16
```

Reply:
Is the text zone large enough to accomodate the longer text?